VotingApplication/VotingApplication.Web.Api/Views/Create/Index.cshtml

```diff
@@ -1,93 +1,167 @@
+                            <input class="checkbox-inline" type="checkbox" id="anonymous-voting">
+                            <a id="anonymous-info" class="help-message">[?]</a>
+                            <div class="tip" data-bind="hoverTargetId: anonymous-info">
+                                <p>Whether poll results should display the voter name, or "Anonymous Voter".</p>
+                                Votes on the Poll Management page will still display names regardless.
+                            </div>
+                        </div>
+
+                        <div id="require-auth-group" class="form-group">
+                            <label class="control-label" for="require-auth">Require Social Media Login</label>
+                            <input class="checkbox-inline" type="checkbox" id="require-auth">
+                            <a id="auth-info" class="help-message">[?]</a>
+                            <div class="tip" data-bind="hoverTargetId: auth-info">
+                                <p>If checked, users will be required to login with Facebook or Google+.</p>
+                                If not checked, users will also have the option to choose their own display name.
+                            </div>
+                        </div>
+
+                        <div id="expiry-voting-group" class="form-group">
+                            <label class="control-label checkbox-label" for="expiry">Time limit</label>
+                            <input class="checkbox-inline" type="checkbox" id="expiry" data-bind="checked: expires">
+                            <a id="timelimit-info" class="help-message">[?]</a>
+                            <div class="tip" data-bind="hoverTargetId: timelimit-info">
+                                If checked, any votes after the expiry date are rejected.
+                            </div>
+                        </div>
+
+                        <div id="expiry-date-group" class="form-group" data-bind="visible: expires">
+                            <label class="control-label" for="expiry-date">Expiry Date</label>
+                            <input class="form-control" id="expiry-date" placeholder="E.g. 01/04/2015 3:14">
+                        </div>
+
+                        <div id="option-adding-group" class="form-group">
+                            <label class="control-label checkbox-label" for="option-adding">Allow users to add options</label>
+                            <input class="checkbox-inline" type="checkbox" id="option-adding">
+                            <a id="add-options-info" class="help-message">[?]</a>
+                            <div class="tip" data-bind="hoverTargetId: add-options-info">
+                                <p>If checked, users will be able to submit their own options from the poll page.</p>
+                                You will still be able to add and remove options from the Poll Management page.
```

Comment:
Some of the descriptions here feel a bit too technical. For example, talking about tokens and explaining that votes given after the expiry date will be rejected.